packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.tsx

```diff
@@ -111,6 +111,7 @@ function _QueryWrapper(props: QueryWrapperProps) {
     nextQueryRequest,
     currentQueryRequest,
     addValueToSelectedFacet,
+    setRangeFacetValue,
```

Comment:
Add to query wrapper to support updating range facets thru query context (just like enum facets)